python/paddle/v2/fluid/backward.py

```diff
@@ -239,7 +304,8 @@ def empty_callback(block, context):
             sub_block = program.block(op.block_attr("sub_block"))
             grad_sub_block = program.create_block(parent_idx=sub_block.idx)
             _append_backward_ops_(sub_block, sub_block.ops, grad_sub_block,
-                                  no_grad_dict, grad_to_var)
+                                  no_grad_dict, grad_to_var,
+                                  _callback_lookup_(op))
```

Comment:
How can we apply more than one callbacks here? E.g. we would like to apply the nccl and error clips at the same time.

And I think `op` is a bad name here. It actually means the op owns the current block. So `op` is too broad.

Reply:
@jiangfeng Thank you for your review.

> How can we apply more than one callbacks here? E.g. we would like to apply the nccl and error clips at the same time.

I will change the callback to a list of callbacks.

> And I think `op` is a bad name. The parameter actually means the `op` owns the current block. `op` is too broad for it.

What do you mean by "The parameter"